hotel/src/chip.rs

```diff
@@ -31,6 +32,8 @@ impl Chip for Hotel {
         unsafe {
             while let Some(nvic_num) = cortexm3::nvic::next_pending() {
                 match nvic_num {
+                    104...109 => crypto::aes::KEYMGR0_AES.handle_interrupt(nvic_num),
```

Comment:
Yeah, I went back and forth on how I felt about this. I like this way better because it doesn't clutter the chip.rs file with unnecessary details. All it knows is that 104 - 109 are for the AES module to deal with.